app/uk/gov/hmrc/individualsemploymentsapi/controller/v2/EmploymentsController.scala

```diff
@@ -84,38 +89,6 @@ abstract class EmploymentsController(employmentsService: EmploymentsService,
 
     } recover withAudit(maybeCorrelationId(request), matchId.toString, "/individuals/employments/paye")
   }
-}
 
-@Singleton
-class SandboxEmploymentsController @Inject()(
-                                              sandboxEmploymentsService: SandboxEmploymentsService,
-                                              scopeService: ScopesService,
-                                              scopesHelper: ScopesHelper,
-                                              scopeFilterVerificationService: ScopeFilterVerificationService,
-                                              val authConnector: AuthConnector,
-                                              @Named("hmctsClientId") val hmctsClientId: String,
-                                              auditHelper: AuditHelper,
-                                              cc: ControllerComponents)(override implicit val ec: ExecutionContext)
-  extends EmploymentsController(sandboxEmploymentsService,
-    scopeService,
-    scopesHelper,
-    scopeFilterVerificationService,
-    auditHelper,
-    cc) {
-  override val environment: String = SANDBOX
-}
-
-@Singleton
-class LiveEmploymentsController @Inject()(
-                                           liveEmploymentsService: LiveEmploymentsService,
-                                           scopeService: ScopesService,
-                                           scopesHelper: ScopesHelper,
-                                           scopeFilterVerificationService: ScopeFilterVerificationService,
-                                           val authConnector: AuthConnector,
-                                           @Named("hmctsClientId") val hmctsClientId: String,
-                                           auditHelper: AuditHelper,
-                                           cc: ControllerComponents)(override implicit val ec: ExecutionContext)
-  extends EmploymentsController(liveEmploymentsService, scopeService, scopesHelper, scopeFilterVerificationService, auditHelper, cc) {
-
-  override val environment: String = PRODUCTION
+  override val environment: String = "NOT_APPLICABLE" // Required due to inheriting common controller and needed for V1
```

Comment:
My preference would be to copy CommonController, so we have a different CommonController for v1 and v2, and remove this. Perhaps get the other dev's opinions?

Reply:
Agreed. We no longer need the environment val.